app/TaxYear/TaxYearServiceProvider.php

```diff
@@ -27,6 +29,11 @@ class TaxYearServiceProvider extends ServiceProvider
      */
     public function register()
     {
+        $this->app->singleton(
+            TaxYearSummaryRepositoryInterface::class,
+            fn (Application $app) => new TaxYearSummaryRepository(),
+        );
```

Comment:
ℹ️ As usual, the domain only holds a repository interface, whose concrete implementation resides in the application layer